runtime/databricks/automl_runtime/forecast/pmdarima/model.py

```diff
@@ -22,26 +22,21 @@
 import pmdarima
 from mlflow.exceptions import MlflowException
 from mlflow.protos.databricks_pb2 import INVALID_PARAMETER_VALUE
+from mlflow.utils.environment import _mlflow_conda_env
 
 from databricks.automl_runtime.forecast import OFFSET_ALIAS_MAP, DATE_OFFSET_KEYWORD_MAP
 from databricks.automl_runtime.forecast.model import ForecastModel, mlflow_forecast_log_model
 from databricks.automl_runtime.forecast.utils import calculate_period_differences, is_frequency_consistency, \
     make_future_dataframe
 
 
-ARIMA_CONDA_ENV = {
-    "channels": ["conda-forge"],
-    "dependencies": [
-        {
-            "pip": [
-                f"pmdarima=={pmdarima.__version__}",
-                f"pickle=={pickle.format_version}",
-                f"pandas=={pd.__version__}",
-            ]
-        }
-    ],
-    "name": "pmdarima_env",
-}
+ARIMA_CONDA_ENV = _mlflow_conda_env(
```

Comment:
@dbczumar to confirm if this is the right function to use. Is there a non-private function that can be used instead of the internal fn?